frontend/public/components/monitoring/receiver-forms/slack-receiver-form.tsx

```diff
@@ -0,0 +1,121 @@
+import * as _ from 'lodash-es';
+import * as React from 'react';
+
+import { SaveAsDefaultCheckbox } from './alert-manager-receiver-forms';
+
+const GLOBAL_FIELDS = ['api_url']; //TODO follow up PR will add advanced fields
+
+// Ex: converts 'client_url' to 'slackClientUrl'
+const getFormFieldName = (propName) =>
+  `slack${_.map(_.split(propName, '_'), (part) => _.upperFirst(part)).join('')}`;
```

Comment:
If we're going to do this, it should be in one place IMO. But I'd rather get away from this entirely.

Reply:
The formFieldNames are different for each of the four receiver forms.

Reply:
Tracking in another comment